src/index.js

```diff
@@ -3,9 +3,36 @@ import movieCounter from './modules/movieCounter.js';
 import './style.css';
 
 const movieList = document.getElementById('movieList');
-displayMovies(movieList);
+displayMovies(movieList, getMovies);
 
 const movieCount = document.getElementById('movie-counter');
 getMovies().then((data) => {
   movieCounter(data, movieCount);
 });
+
+const startIndex = document.getElementById('startIndex');
+const endIndex = document.getElementById('endIndex');
+const showMovies = document.getElementById('showMovies');
+const errorMsg = document.getElementById('error');
+
+showMovies.onclick = () => {
+  const start = startIndex.value;
+  const end = endIndex.value;
+  if (start !== '' && end !== '') {
+    if (+start >= +end) {
+      errorMsg.innerHTML = 'Invalid input';
+    }
+    displayMovies(movieList, () => getMovies(start, end));
+    getMovies(start, end).then((data) => {
+      movieCounter(data, movieCount);
```

Comment:
Good job in implementing this feature on your page. This will enhance the user experience. 
I suggest you to add this key feature in your Read me file and update the same.